packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -179,6 +180,11 @@ export default class ImageEdit implements EditorPlugin {
                     e.preventDefault();
                 }
             },
+            mouseup: e => {
```

Comment:
we already have mouse up event handler at line 185

Reply:
The handler in the switch case is not triggered when the mouse up event happens in the image wrapper.

Reply:
I see. In that case, we should only attach this event handler when image wrapper is showing, and detach it when hide.

Also please try if we can attach the event to the wrapper element only, but not the whole window.

Reply:
Unless the user always have the cursor on the handles during the whole rotate and resize process, the event will not be triggered. Sometimes the cursor can move out from the handles. 

![resizeAndRotate](https://github.com/microsoft/roosterjs/assets/87443959/6fed9e70-0a06-45db-8c96-d8c88d60ddf9)

Reply:
Try pass true as the 3rd parameter when addEventListener, see if that can fix this problem:

```ts
node.addEventListener('mouseup', handler, true /*useCapture*/);
```

If that works, also need to pass true when remove event listener.